eagleproject/core/templates/analytics_report_email.html

```diff
@@ -1,150 +1,209 @@
 {% load blockchain %}
 
+<head>
+  <link href="/static/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+</head>
+
 <div>
-    <style>
-        h2 {
-            margin-bottom: 20px;
-        }
-
-        .table-header: {
-            background-color: #e5e5e5;
-            border: 1px solid black;
-        }
-
-        .green {
-            color: green;
-        }
-
-        .red {
-            color: red;
-        }
-    </style>
-
-    <h2>{{type}} Analytics Report</h2>
-    <h4>
+  <div class="box">
+    <img src="/static/core/img/origin-dollar.svg" alt="origin dollar" class="wordmark" />
```

Comment:
Should these be absolute URLs? I don't think relative URLs would work (unless Django does something when using a template)

Reply:
Also, even for absolute URLs, if the host of the images and files are different from the email domain, it might get flagged as spam/junk. It's probably worth testing these emails (using a SendGrid API key or something) before we merge it in